build/pipeline-ci.yaml

```diff
@@ -17,37 +17,11 @@ pr: none
 variables:
   BuildPlatform: 'any cpu'
   BuildConfiguration: 'debug'
+  FastRun: true
```

Comment:
not a fan of this concept. we should always be fast. I prefer flags like "skip tests"